foundry.toml

```diff
@@ -14,24 +14,24 @@ cache = true
 cache_path = 'forge-cache'
 # Import statement remappings
 remappings = [
-  'solady=lib/solady/src',
-  'openzeppelin=lib/openzeppelin-contracts/contracts',
+  'forge-std=lib/forge-std/src',
   'hyperdrive=lib/hyperdrive',
+  'openzeppelin=lib/openzeppelin-contracts/contracts',
+  'solady=lib/solady/src',
 ]
 # gas limit - max u64
 gas_limit = "18446744073709551615"
 # allows the ffi to be used
 ffi = true
 # strict warnings
-deny_warnings = true
+# deny_warnings = true
```

Comment:
Is the plan to reenable this? If so, it would probably be good to leave a TODO or FIXME.